src/components/PageLayout/PageLayout_View.tsx

```diff
@@ -14,11 +14,12 @@ const navigation = [
   { name: "Markets", path: "/markets" },
   { name: "Bets", path: "/bets" },
   //{ name: "HL Token", path: "/tokens" },
-  {
+/**  {
```

Comment:
do we not want to remove it rather than comment out?

Reply:
commented out the tokens page, so though we'd do the same for this